### DISCUSSION TITLE
求xray的API命令行参数adi、ado、rmi、rmo详解

### DISCUSSION THREAD

**Discussion by a108393** [XTLS/Xray-core#1084] [Q&A]
<img width="778" alt="WechatIMG1383" src="https://user-images.githubusercontent.com/91522471/170838861-fe54047c-af28-46ef-b808-25e9ffe1682c.png">

版本/version: xray 1.4.2
操作系统/OS: centos
我想通过command命令来操作xray，请求帮助。
譬如  xray api adi [--server=127.0.0.1:8080] <c1.json> [c2.json]...
c1.json和c2.json怎么写？

求api下的adi、ado、rmi、rmo的json详细的参数示例！感谢！！

**Comment by tsosunchia** [comment]
adi           Add inbounds
	ado           Add outbounds
	rmi           Remove inbounds
	rmo           Remove outbounds

**Comment by a108393** [comment]
@tsosunchia  xray api adi [--server=127.0.0.1:8080] <c1.json> [c2.json]...
c1.json和c2.json怎么写？

主要想知道c1.json和c2.json怎么写？感谢！请问能不能给个示例？
c1.json、c2.json的内容和格式？
